cartography/intel/aws/ec2/network_interfaces.py

```diff
@@ -188,56 +188,57 @@ def load_network_interface_elbv2_relations(
 
 
 @timeit
-def load_network_interface_instance_to_subnet_relations(neo4j_session: neo4j.Session, update_tag: int) -> None:
+def load_network_interface_instance_to_subnet_relations(neo4j_session: neo4j.Session, aws_account_id, update_tag: int) -> None:
     """
     Creates (:EC2Instance)-[:PART_OF_SUBNET]->(:EC2Subnet) if
     (:EC2Instance)--(:NetworkInterface)--(:EC2Subnet).
     """
     ingest_network_interface_instance_relations = """
-    MATCH (i:EC2Instance)-[:NETWORK_INTERFACE]-(interface:NetworkInterface)-[:PART_OF_SUBNET]-(s:EC2Subnet)
+    MATCH(aa: AWSAccount{id: $AWS_ACCOUNT_ID}), (i:EC2Instance)-[:NETWORK_INTERFACE]-(interface:NetworkInterface)-[:PART_OF_SUBNET]-(s:EC2Subnet)
```

Comment:
this query is incorrect, we should check for the relationship between `AWSAccount` and `EC2Instance` so that we know the instances which are picked belong to the aws account